src/datapump/sync/sync.py

```diff
@@ -377,67 +402,179 @@ def build_jobs(self, config: DatapumpConfig) -> List[Job]:
                 resampling="nearest",
                 symbology={"type": "date_conf_intensity"},
             ),
-            content_date_range=ContentDateRange(
-                min="2014-12-31", max=str(latest_release_dt)
-            ),
+            content_date_range=ContentDateRange(min="2014-12-31", max=str(version_dt)),
```

Comment:
is start date the same for all alert types?

Reply:
Good point! For some reason RADD had already it at 2014-12-31, even though it start in 2020. I guess I'd want to check with @dmannarino if there was a reason he had it this way (e.g. flagship needed them all to have the same start time).

Reply:
Hmm, I'm trying to remember what the front-end uses it for. If they use it for determining which alerts fall within the 2-year window, they may need it to be 2014-12-31. If so perhaps they should hard-code it as a reference value in the layer/decode function so we can set this appropriately per-dataset.

Reply:
hmm ok I'll leave it as 2014-12-31 for now since that works for RADD, and we can follow up with Vizz to see how they use it